services/site/src/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -27,7 +27,7 @@ const Breadcrumbs: React.FunctionComponent = () => {
     return () => {
       isMounted = false;
     };
-  }, [apolloClient, router, router.pathname]);
+  }, [apolloClient, pathname, JSON.stringify(params)]);
```

Comment:
`useParams` returns a new object each time it's called. Using that as a trigger for an effect leads to a rendering infinity loop. As a fix we JSON encode the object so that the value changes only if one of the values changes.